src/redux/reducers/todo.reducer.js

```diff
@@ -0,0 +1,17 @@
+import * as AT from '../action.types';
+
+const INITIAL_STATE = {dataStream: {label: 'Instagram'}, rangeDate: {startDate: {}, endDate: {}}};
+const todoReducer = (state = INITIAL_STATE, action) => {
+	switch (action.type) {
+		case AT.DATA_STREAM_REQUEST:
+			console.log(action.payload);
+			return {...state, dataStream: action.payload.dataStream};
```

Comment:
here you can use set from lodash

import {set} from 'lodash/fp';

and - instead of return {...state, dataStream: action.payload.dataStream};
do - return set('dataStream', action.payload.dataStream, state);

set will return an immutable data